libs/core/src/components/pds-checkbox/pds-checkbox.scss

```diff
@@ -1,21 +1,21 @@
 :host {
-  --background-color: var(--pine-color-base-white);
-  --background-color-disabled: var(--pine-color-neutral-grey-200);
-  --background-color-hover: var(--pine-color-neutral-grey-100);
-  --border: var(--pine-border-interactive);
-  --border-color-disabled: var(--pine-color-neutral-charcoal-100);
-  --border-color-hover: var(--pine-border-interactive-hover-color);
-  --border-color-icon: var(--pine-color-base-white);
-  --border-radius: var(--pine-border-radius-xs);
+  --border: var(--pine-border-width-thin) solid var(--pine-color-grey-400);
+  --border-color-disabled: var(--pine-color-charcoal-100);
+  --border-color-icon: var(--pine-color-white);
+  --border-color-interactive-hover: var(--pine-border-width-thin) solid var(--pine-color-grey-500);
+  --border-radius: var(--pine-border-radius-050);
   --box-shadow-focus: 0 0 0 2px var(--pine-color-primary-200);
   --box-shadow-focus-invalid: 0 0 0 2px var(--pine-color-red-200);
-  --color-checked: var(--pine-color-neutral-charcoal-500);
-  --color-disabled: var(--pine-color-neutral-grey-300);
+  --color-background: var(--pine-color-white);
+  --color-background-disabled: var(--pine-color-grey-200);
+  --color-background-hover: var(--pine-color-grey-100);
+  --color-checked: var(--pine-color-charcoal-500);
+  --color-disabled: var(--pine-color-grey-300);
   --color-focus: var(--pine-color-primary-200);
-  --color-invalid-focus: var(--pine-color-red-200);
   --color-invalid: var(--pine-color-red-300);
-  --message-color-disabled: var(--pine-color-neutral-grey-500);
-  --message-color: var(--pine-color-neutral-charcoal-200);
+  --color-invalid-focus: var(--pine-color-red-200);
```

Comment:
Can we add `text` in here? Would help give better context on where that color is being applied.

Reply:
the color-invalid applies to more than just any text, but let me know if I am misunderstanding.

Reply:
@pixelflips Hmm yep, I see your point and angle here. Originally, I intended to be more explicit with how/where certain tokens will be used but it does bring up the point about cases where some may/could be reused. I think I may have done this in a few places as well. I'd like to see what the team thinks about it overall, but let's get this in for now.